### ISSUE TITLE
7.4.8. Screen Name/Title: Browse templates – Filter Panel - Non-Keyboard-Focusable Filter Links

### ISSUE THREAD

**Issue by whabanks** [cds-snc/notification-planning#2341]
**Component:** 
Apply filters

**Observation:**
Non-Keyboard-Focusable Filter Links
The filter options for “Type” and “Category” (e.g., “Email”, “Text message”, “Confirmation”, etc.) are implemented as `<a>` elements with tabindex="-1". This removes them from the tab order, making them inaccessible to keyboard-only users.
This issue blocks interaction for keyboard users, making filter functionality inaccessible unless a mouse or other pointer is used.

**Recommendation:** 
Remove tabindex="-1" from the `<a>` elements or replace them with `<button>` elements if they perform an action (e.g., filtering content). Ensure they are part of the natural tab order.

**Comment by smcmurtry** [comment]
This looks like a bug, maybe browser specific. Watch the attached video and we should try to reproduce. Maybe related to the details polyfill.

**Comment by whabanks** [comment]
This doesn't appear to be an issue or may have been fixed with the [removal of the details polyfill](https://github.com/cds-snc/notification-admin/pull/2379/files). Unable to reproduce in both Chrome and Firefox:

https://github.com/user-attachments/assets/88382742-1d62-4c1c-bc7c-030f5db3f7ce
